vj4/model/document.py

```diff
@@ -60,7 +62,7 @@ def convert_doc_id(doc_id):
 
 
 @argmethod.wrap
-async def get(domain_id: str, doc_type: int, doc_id: convert_doc_id, fields=None):
+async def get(domain_id: str, doc_type: Union[int, Dict], doc_id: convert_doc_id, fields=None):
```

Comment:
vj4 does not use pytype annotation. the annotation here is for `argmethod.wrap` to parse command line args, and `Union` does not work.